CHANGELOG.rst

```diff
@@ -14,6 +14,14 @@ Change Log
 Unreleased
 **********
 
+[4.0.0] - 2023-04-26
+********************
+Changed
+=======
+* Implement openedx-events Event Bus Consumer API
+* **BREAKING CHANGE**: Remove ``consume_events`` management command as this
+  command will be provided by openedx_events.
```

Comment:
```suggestion
* **BREAKING CHANGE**: Remove ``consume_events`` management command as this
  command will be provided by openedx_events. Resetting offsets now requires passing `--extra '{"offset_time": "2023-01-08T06:46:22"}'` rather than `-o 2023-01-08T06:46:22`.
```

Reply:
There should also be some mention of the changes to the constructor and method signatures of `KafkaEventConsumer` in case there are still direct references in relying code.

Reply:
Updated!